src/likelihoodratiotest.jl

```diff
@@ -215,3 +255,49 @@ function StatsModels.isnested(m1::MixedModel, m2::MixedModel; atol::Real=0.0)
 
     true
 end
+
+
+"""
+    isnested(m1::LinearModel, m2::LinearMixedModel; atol::Real=0.0)
+    isnested(m1::TableRegressioLinearModel{LinearModel}, m2::LinearMixedModel; atol::Real=0.0)
+    isnested(m1::GeneralizedLinearModel, m2::GeneralizedLinearMixedModel; atol::Real=0.0)
+    isnested(m1::TableRegressioLinearModel{GeneralizedLinearModel}, m2::GeneralizedLinearMixedModel; atol::Real=0.0)
+
+Indicate whether model `m1` is nested in model `m2`, i.e. whether
+`m1` can be obtained by constraining some parameters in `m2`.
+Both models must have been fitted on the same data. The mixed model must be
+fitted with maximum likelihood (i.e. **not** REML).
+"""
+function StatsModels.isnested(m1::TableRegressionModel{<:Union{LinearModel, GeneralizedLinearModel}},
+                              m2::MixedModel; atol::Real=0.0)
+    StatsModels.isnested(m1.model, m2) || return false
+
+    # check that the nested fixef are a subset of the outer
+    all(in.(coefnames(m1),  Ref(coefnames(m2)))) || return false
```

Comment:
I think this has a curried form:
```suggestion
    all(in(coefnames(m2)), coefnames(m1)) || return false
```

Reply:
Apparently, it does!

That saves a small allocation, but makes it less readable for me.  But good to know!